npc/mapflag/noteleport.txt

```diff
@@ -154,6 +154,7 @@ thor_camp	mapflag	noteleport
 moc_para01	mapflag	noteleport
 
 // Job Quests ====================
+pay_arche	mapflag	noteleport
```

Comment:
this change look like unrelated to messages fixes

Reply:
if you want add map flags, do this in separate commit

Reply:
Fixed.